scripts/copy-res.js

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env node
+
+// copies resources into the lib directory.
+
+const parseArgs = require('minimist');
+const chokidar = require('chokidar');
+const path = require('path');
+const fs = require('fs');
+
+const argv = parseArgs(process.argv.slice(2), {});
+
+const watch = argv.w;
```

Comment:
Unclear if anyone will know the watching is here if it's not exposed by `package.json` scripts... 🤷

Reply:
Happy to get rid of it to minimise this copy-res script further. I didn't wire up watching to `start` as the Electron app wouldn't pick it up without restarting anyhow